### ISSUE TITLE
Issue with building and installing

### ISSUE THREAD

**Issue by Karma2424** [marcograss/partialzip#10]
Can't install partialzip after 3.3.0 from cargo or build it from source getting this error:

```
error: failed to compile `partialzip v4.0.0`, intermediate artifacts can be found at `/tmp/cargo-installb3lTSf`

Caused by:
  failed to select a version for `env_logger`.
      ... required by package `partialzip v4.0.0`
  versions that meet the requirements `^0.11.5` are: 0.11.5

  the package `partialzip` depends on `env_logger`, with features: `anstream` but `env_logger` does not have these features.
   It has an optional dependency with that name, but that dependency uses the "dep:" syntax in the features table, so it does not have an implicit feature with that name.


  failed to select a version for `env_logger` which could resolve this conflict
```

**Comment by Karma2424** [comment]
TLDR: Install Rust using rustup instead of apt, as the version from Debian is 1.63.0, while the latest from rustup is 1.80.1. I got this info from alucryd/oxyromon#130 and sorry 😅 marcograss for the unnecessary issue.

**Comment by marcograss** [comment]
ops no problem yeah, unfortunately debian and ubuntu are plagued by very old packages
